build.cake

```diff
@@ -70,21 +70,21 @@ Task("ReplaceStubWithExample")
         }
     });
 
-Task("TestUsingExampleImplementation")
-    .IsDependentOn("ReplaceStubWithExample")
-    .Does(() => {
-        var allProjects = GetFiles(exercisesBuildDir + "/*/*.csproj");
-        Parallel.ForEach(allProjects, parallelOptions, (project) => DotNetCoreTest(project.FullPath));
-    });
 
 Task("BuildGenerators")
     .Does(() => {
        DotNetCoreBuild("./generators/Generators.csproj");
     });
 
+Task("TestUsingExampleImplementation")
```

Comment:
I understand the idea of moving this task below the BuildGenerators task, but I'd rather the BuildGenerators task be placed higher in the build script as the TestUsingExampleImplementation task is dependent on the tasks above it, and I'd prefer keeping them listed sequentially.

Reply:
Agree, fixed